src/components/buttonMenu/buttonMenu.stories.svelte

```diff
@@ -26,12 +29,13 @@
       </leo-menu-item>
       <div class="custom-item">
         <div>Suggested questions</div>
-        <Toggle size="small" />
+        <Toggle checked={toggleIsChecked} size="small" />
```

Comment:
Should this be 
```suggestion
        <Toggle bind:checked={toggleIsChecked} size="small" />
```
so toggling it updates the variable & vice versa?

Reply:
good spot - fixed